systemd/dlt.service.cmake

```diff
@@ -24,6 +26,3 @@ ExecStart=@CMAKE_INSTALL_PREFIX@/bin/dlt-daemon
 WatchdogSec=@DLT_WatchdogSec@
 NotifyAccess=main
 LimitCORE=infinity
-
-[Install]
-WantedBy=basic.target
```

Comment:
Hello @minminlittleshrimp ,
this disables the start of dlt. Did you want to shift this one also to multi-user.target?

Reply:
Hello @michael-methner 
This should not be shifted since daemon should be up before dlt-system or any in multi-user.
I will correct it.